biopet-utils/src/main/scala/nl/lumc/sasc/biopet/utils/implicits/Either.scala

```diff
@@ -0,0 +1,40 @@
+package nl.lumc.sasc.biopet.utils.implicits
+
+/**
+ * Created by pjvanthof on 20/02/2017.
+ */
+object Either {
```

Comment:
Great. For the record, scala'z `Either` is called `\/`. It's not exactly pronounceable, but the official term is disjunction (like the mathematical disjunction symbol ν, which is sort of a fancy name for 'or'.

Anyway, look for `Option.to[Right|Left]Disjunction` methods there.